### DISCUSSION TITLE
Debugging 413 responses in production on Vercel

### DISCUSSION THREAD

**Discussion by good-idea** [vercel/next.js#36633] [Help]
I have a deployment that is showing 413 errors in the function logs when fetching particular routes & next JSON data.

Oddly, this only happens occasionally - see the example logs below:

(fetching the page JSON)
<img width="1021" alt="image" src="https://user-images.githubusercontent.com/11514928/166344928-5ddb7f04-262b-4ee2-8c3a-28bb4158ec89.png">

(fetching the page HTML)
<img width="1028" alt="image" src="https://user-images.githubusercontent.com/11514928/166345097-1480daee-454f-42ee-9361-a8a64eae761d.png">

Oddly, two requests show here, one that fails and one that does not. The payload size for under the 1mb limit for vercel. What else could be triggering this error?

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/11514928/166344904-0252cf6b-0c49-4739-a74d-47ce2dfb8848.png">

**Comment by icyJoseph** [comment]
Could it be that somehow, a request inside the GSSP fails? and the handler just never ends? That it happens occasionally makes me think that an async process fails, and there's no fallback for it.
